book/src/overview.md

```diff
@@ -140,7 +143,7 @@ The algorithm Salsa uses to decide when a tracked function needs to be re-execut
 Tracked functions have to follow a particular structure:
 
 - They must take a `&`-reference to the database as their first argument.
-  - Note that because this is an `&`-reference, it is not possible to create or modify inputs during a tracked function!
```

Comment:
It does seem a little unfortunate that inputs can be created in tracked functions now. Although since you still need `&mut` to modify an input, I guess it doesn't really matter because they'll basically behave like tracked structs.

Reply:
I think that was intentional -- so that you can have lazilly created inputs

Reply:
Ah, true.